media-libs/dav1d/dav1d-1.1.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
`pkgcheck scan --commits` should catch that this is wrong, needs to be 2023. Fixed on merge